src/content/docs/configuration/automount_with_fstab.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Automount Additional Drives Through fstab at Boot
+description: Mount additional static drives at boot by utilizing the file found at /etc/fstab
+---
+
+This tutorial will describe the basics of utilizing the fstab file located in /etc/ in order to mount static drives during boot. It will briefly explain how to find a partition or drive's UUID, what some options do, and further reading should the information provided be insufficient.
+
+## Prerequisites
+- Root access
+
+## Adding Entries to /etc/fstab
+
+*Click [here](automount_with_fstab#tldr) to skip to the tl;dr*
+
+### 1. Creating the Mount Location
+
+Before we can mount our drive with fstab, or ***f***ile ***s***ystems ***tab***le, we will need to create a mount point for our drive.
+
+The [Linux Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html) says that `/media/` is the proper location for removable drives to be mounted. For this reason we will create a new directory to mount our drive to inside of the `/media/` directory. In this example we will be mounting a Windows partition. Despite that though these same instructions apply universally, with rare exceptions, such as a NAS or a btrfs subvolume. 
+
+Create the mount point:
+
+```
+sudo mkdir /media/windows
+```
+
+Because `/media/` is owned by root, we need to run this command as root to create the mount point.
```

Comment:
This is a redundant step as if a mount point doesn't exist, it will just create one.